docs/proposals/kubelet-oci-runtime.md

```diff
@@ -0,0 +1,87 @@
+Kubelet OCI runtime integration
+===========================================================
+Authors: Mrunal Patel (@mrunalp), Vishnu Kannan (@vishh)
+
+## Abstract
+This proposal aims to introduce support for OCI Runtime Specification compatible runtimes in Kubelet.
+
+## Motivation
+Kubelet should support runc and OCI as a container runtime to support Open Standards. Runc is the OCI compliant
+implementation of the runtime specification for Linux. It has all the features in the OCI Runtime Spec, as well as
+runtime options to enable additional features. It should be possible to support pods using runc as it has all the
+primitives to support pods. In addition to runc, compatibility with OCI Specification will let kubernetes support other
```

Comment:
I think this needs to be clarified as well. The primitives runc has to support pods is the ability to join namespaces of other processes. There are lots of open questions here still such as how do PID namespaces work and how does the creation and deletion of namespaces get orchestrated. All of these things are left as implementation details to the orchestrator.